### ISSUE TITLE
ublock origin disabled in Chrome

### ISSUE THREAD

**Issue by silat** [gorhill/uBlock#285]
ublock origin 982.
It keeps disabling in Chrome v 43.0.2357.81 m.
I have to re-enable each and every restart.

**Comment by CrisBRM** [comment]
Something's wrong with your Chromium installation. Has happened to me before (use firefox now). Next time you feel like updating chroming, delete the "Application" folder before installing.

**Comment by gorhill** [comment]
I think you should try this: https://adblockplus.org/forum/viewtopic.php?p=102370

**Comment by gorhill** [comment]
@silat From where did you install your uBlock?

**Comment by CrisBRM** [comment]
@gorhill I think the issue is that Chrom* is not saving his settings due to a corrupt installation. Has happened to me before and I had to re-enable uBlock on a per session basis, as well as re-position the window every single time.

**Comment by silat** [comment]
Hi,
  I installed from the Chrome Store for Chrome Chrome store
<https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm>
and
FF FF Addons
<https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/?src=search>

The problem was on: Chrome stable and Chrome Canary.
I finally uninstalled the extensions in both browsers from the extension
page and cleaned out any remnants in the Canary and Stable folders for the
5th time and it is now working :)

They both updated today and all is still working properly.
Sorry for any trouble I cause you.

**Comment by silat** [comment]
Chris thanks for your help.
 I installed from the Chrome Store for Chrome Chrome store
<https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm>
and
FF FF Addons
<https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/?src=search>

The problem was on: Chrome stable and Chrome Canary.
I finally uninstalled the extensions in both browsers from the extension
page and cleaned out any remnants in the Canary and Stable folders for the
5th time and it is now working :)

They both updated today and all is still working properly.
Sorry for any trouble I cause you.
